src/schema/comments.ts

```diff
@@ -549,12 +549,17 @@ const getCommentById = async (
   ctx: Context,
   info: GraphQLResolveInfo,
 ): Promise<GQLComment> => {
-  const res = await graphorm.query<GQLComment>(ctx, info, (builder) => {
-    builder.queryBuilder = builder.queryBuilder
-      .andWhere(`${builder.alias}.id = :id`, { id })
-      .limit(1);
-    return builder;
-  });
+  const res = await graphorm.query<GQLComment>(
```

Comment:
So here we can't use read replica because you are trying to return modified data